tests/debugpy/test_evaluate.py

```diff
@@ -677,3 +677,88 @@ def code_to_debug():
 
         session.request_continue()
         assert backchannel.receive() == 1000
+
+
+def test_evaluate_thread_locks(pyfile, target, run):
+    @pyfile
+    def code_to_debug():
+        """
+        The idea here is that a secondary thread does the processing of instructions,
+        so, when all threads are stopped, doing an evaluation for:
+
+        processor.process('xxx')
+
+        would be locked until secondary threads start running.
+        See: https://github.com/microsoft/debugpy/issues/157
+        """
+
+        import threading
+
+        try:
+            from queue import Queue
+        except:
+            from Queue import Queue
```

Comment:
This is handled by `compat` - it exposes it as `queue`. Since this is an stdlib module, it's okay to use it unqualified after doing `import queue from debugpy.common.compat`.